avro4s-core/src/test/scala/com/sksamuel/avro4s/schema/DateSchemaTest.scala

```diff
@@ -30,9 +30,9 @@ class DateSchemaTest extends FunSuite with Matchers {
   }
 
   test("generate time logical type for LocalDateTime") {
-    case class LocalTimeTest(time: LocalTime)
+    case class LocalDateTimeTest(time: LocalDateTime)
     val expected = new org.apache.avro.Schema.Parser().parse(getClass.getResourceAsStream("/localdatetime.json"))
-    val schema = AvroSchema[LocalTimeTest]
+    val schema = AvroSchema[LocalDateTimeTest]
     schema.toString(true) shouldBe expected.toString(true)
```

Comment:
When I fixed the schema test for LocalTime, I noticed the test for LocalDateTime was the test for LocalTime copied and pasted.  I fixed this test as well.